liquibase-core/src/main/java/liquibase/parser/ChangeLogParserConfiguration.java

```diff
@@ -41,6 +41,10 @@ public class ChangeLogParserConfiguration implements AutoloadedConfigurations {
                 .setDefaultValue(ChangelogParseMode.STRICT)
                 .build();
 
+        WARN_ON_MISSING_CHANGELOGS = builder.define("warnOnMissingChangeLogs", Boolean.class)
```

Comment:
Would you mind to also rename this variable to `WARN_OR_ERROR_ON_MISSING_CHANGELOGS`? Since if the default value set is `true` it will return an error, instead of a warning.